config/environments/development.rb

```diff
@@ -38,4 +38,6 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.action_mailer.default_options = { from: ENV['EMAIL_DEFAULT'] }
+  config.action_mailer.default_url_options = { host: ENV['URL_DEFAULT'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.